backend/spec/features/admin/orders/order_details_spec.rb

```diff
@@ -530,4 +475,23 @@
       end
     end
   end
+
+  context 'with only read permissions' do
+    stub_authorization!(permission_sets: [Spree::PermissionSets::OrderDisplay])
+
+    it "should not display forbidden links" do
```

Comment:
You consistently use single quotes in the literal expectations but not in the description? Can we sync these up?